api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -183,18 +181,21 @@ public long getConceptId() {
     Clock clock() {
       return CLOCK;
     }
+
+    @Bean
+    DbUser user() {
+      return user;
+    }
   }
 
   @Before
   public void setUp() {
-    DbUser user = new DbUser();
+    user = new DbUser();
```

Comment:
BTW, If you wanted, because the user object is static, you could use a `@BeforeClass` method and only initialize it once. I'm not sure that's a win, though, since you're doing dao things.

Reply:
I generally prefer to put setup stuff in `Before` instead of `BeforeClass` so each method can start with the same clean state.

`BeforeClass` might be a good choice if I'm constructing something really heavy or immutable.

In this case, mock models are meant to be modified as needed so we need to reinitialize the model or modifications from a previous test will bleed into the following.